src/vision.cs

```diff
@@ -1891,6 +2145,11 @@ interface VNRectangleObservation {
 		[Static]
 		[Export ("observationWithRequestRevision:boundingBox:")]
 		VNRectangleObservation FromBoundingBox (VNRectangleObservationRequestRevision requestRevision, CGRect boundingBox);
+
+		[TV(17, 0), Mac(14, 0), iOS(17, 0), MacCatalyst(17,0)]
+		[Static]
+		[Export("rectangleObservationWithRequestRevision:topLeft:topRight:bottomRight:bottomLeft:")]
+		VNRectangleObservation GetRectangleObservation (nuint requestRevision, CGPoint topLeft, CGPoint topRight, CGPoint bottomRight, CGPoint bottomLeft);
```

Comment:
The other static methods are using the From* pattern. We probably want all the selectors to use the same naming.